src/desktop/components/main_layout/templates/index.jade

```diff
@@ -9,7 +9,7 @@ html(
   data-useragent=userAgent
   data-user-type=sd.CURRENT_USER && sd.CURRENT_USER.type
   data-lab-features=sd.CURRENT_USER && sd.CURRENT_USER.lab_features && sd.CURRENT_USER.lab_features.join(',')
-  data-autosuggest-search=sd.AUTOSUGGEST_SEARCH
+  data-autosuggest-search=!(sd.CURRENT_USER && sd.CURRENT_USER.lab_features && sd.CURRENT_USER.lab_features.indexOf('Grouped Search Results') != -1) && sd.AUTOSUGGEST_SEARCH === 'experiment'
```

Comment:
We want to add some custom CSS if youre in the experiment spotlight test group..._and dont have this new lab on_

Reply:
this is starting to get uncomfortably complex

Reply:
yea...i dont think its _that_ bad, but its getting rough.